### DISCUSSION TITLE
The "free" version feels like visiting a mainstream media page without an adblocker

### DISCUSSION THREAD

**Discussion by Leonetienne** [chevereto/chevereto#79] [General]
where's the point in self-hosting it then, anyway. rm'ed it gets...

**Comment by rodber** [comment]
Hello,

Just to clarify, there are no ads in Chevereto free edition, so it’s hard to understand how it could feel that way. I assume you’re referring to the “$Pro” links in the menu. If that’s the case, you might want to check out this discussion: https://github.com/chevereto/chevereto/issues/77.

I’ve actively worked on removing certain paid prompts, allowing logo edits, and supporting external storage servers—it’s all there in the issues section for transparency.

But I have to say, your comment makes me rethink those efforts. For you, this might just be another software you stumbled across yesterday, but for me, this is a product I’ve been building since 2007—it’s my life’s work.

Thanks for sharing your thoughts.

**Comment by Leonetienne** [comment]
Thanks for your reply. What i mean is that the free version is littered with calls-to-actions which all prompt the user to buy a (btw rly expensive license. it's half phpstorms cost). And that in combination with the fact the free versions public frontend is basically a giant billboard for chevereto.com, with some user-defined pictures. I would not say that the free version is a viable tool to share photography for the average self-hoster because of that.

> But I have to say, your comment makes me rethink those efforts. For you, this might just be another software you stumbled across yesterday, but for me, this is a product I’ve been building since 2007—it’s my life’s work.

yes, exactly. I just discovered this last night and spent 3am to 4:30 am getting it to run just to discover, i basically can't do anything with it, because it doesn't allow me to. That made me a bit upset and i am sorry for that.

**Comment by rodber** [comment]
Thanks for your feedback, and I understand your frustration.

For context, all the issues you mentioned regarding prompts and limitations are already being considered and discussed in our issue tracker. If you're interested, you can review those ongoing discussions here: https://github.com/chevereto/chevereto/issues.

**Comment by rodber** [comment]
https://github.com/chevereto/chevereto/releases/tag/4.2.0

**Comment by antropolouco** [comment]
Hey there, @rodber !

I just sent two pull requests, updating pt* translation files.

I'm writing here to let you know that I do appreciate your work, and I hope the free version can be mantained and expanded. I've recently installed Chevereto over other options, for securely (in our case, privately) keeping and sharing memories related to an yet-to-be-born child, so the child will have the chance to easily access all their own history during the time and deep in the following years. I hope you consider support from crazy people like me, when it comes to rethinking your efforts with the free version.

Perhaps the person here felt this way because of the lack for documentation(?) explaining, for example, the free version restrictions in comparison to PRO or whatever - perhaps the person was expecting the full-power advertised by the project (as I also was, in the first place). I must confess I was a little disappointed when I found out I couldn't simply register further user accounts, being restricted to the sole/admin account (something I discovered just after installing it on my server). I enjoy the app despite of that, because it clearly fills my own, concrete, demand. So you can count on me for improving the docs and making it more clear for newcomers which features/specs/limitations they'll get with the free version.

Saludos, hermanito &
Best regards

**Comment by rodber** [comment]
Hello! @antropolouco

> I just sent two pull requests, updating pt* translation files.

Many thanks for your contribution! The updated `pt*` files will be included in v4.2.3.

> Perhaps the person here felt this way because of the lack for documentation(?)

I believe the issue might be that the comparison list in `README.md` includes too many features shared among all editions, which can be confusing for users. A summary or table with the specific features included in each edition would help make comparisons clearer and avoid any surprises.

The feature set is always evolving. As the project matures, more features are added to the `chevereto/chevereto` base. For example, I recently opened logo editing and storage APIs for v4.2, and I just [announced](https://github.com/chevereto/chevereto/discussions/88) that the Bulk Importer will also be included. I wish the project could progress faster and include more features over time, but it can be challenging.

Many thanks for your support!
Saludos desde Chile

**Reply by antropolouco** [reply]
> I believe the issue might be that the comparison list in `README.md` includes too many features shared among all editions, which can be confusing for users. A summary or table with the specific features included in each edition would help make comparisons clearer and avoid any surprises.

Definitely. Let me know if/how I can help you with this.
 
> The feature set is always evolving. As the project matures, more features are added to the `chevereto/chevereto` base. For example, I recently opened logo editing and storage APIs for v4.2, and I just [announced](https://github.com/chevereto/chevereto/discussions/88) that the Bulk Importer will also be included. I wish the project could progress faster and include more features over time, but it can be challenging.

Yeah, I saw the recent features, which facilitate a lot for everybody to make Chevereto theirs, thank you very much for that. Once again, thank you for your work and count on me for supporting the project - which I'll likely offer/recommend as a solution for other persons.

Saludos desde Brasil y Guatemala jajajajaja :wink:
